client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx

```diff
@@ -2,24 +2,20 @@ import classNames from 'classnames'
 import React from 'react'
 
 import { DismissibleAlert } from '@sourcegraph/web/src/components/DismissibleAlert'
-import { CardBody, Card, Link } from '@sourcegraph/wildcard'
+import { CardBody, Card } from '@sourcegraph/wildcard'
 
 import styles from './BatchChangesListIntro.module.scss'
 
 export const BatchChangesChangelogAlert: React.FunctionComponent = () => (
     <DismissibleAlert
         className={styles.batchChangesListIntroAlert}
-        partialStorageKey="batch-changes-list-intro-changelog-3.36"
+        partialStorageKey="batch-changes-list-intro-changelog-3.37"
     >
         <Card className={classNames(styles.batchChangesListIntroCard, 'h-100')}>
             <CardBody>
-                <h4>Batch Changes updates in version 3.36</h4>
+                <h4>Batch Changes updates in version 3.37</h4>
                 <ul className="mb-0 pl-3">
-                    <li>
-                        <Link to="/help/admin/config/batch_changes#forks" rel="noopener" target="_blank">
-                            Batch Changes now supports pushing changesets to forked repositories.
-                        </Link>
-                    </li>
+                    <li>Nothing noteworthy this time.</li>
```

Comment:
Aw man, put this way, it sounds so depressing. 😂 Could we allude to upcoming changes instead, maybe? This is probably way too cheesy but:
```suggestion
                    <li>Nothing noteworthy this time, but rest assured, there's some big stuff cooking.</li>
```

Reply:
It's not actually rendered in the app, so I guess it doesn't matter?

Reply:
Ohhh I see, we *only* use it in `BatchChangesListIntro`? Yeah don't worry about it then. 😂